app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17, \
+        "test sum of the parts should be equal to value"
```

Comment:
Try not to use `\` as a line breaker, you can use parentheses instead

Reply:
Hi, when I use parentheses i get this type of error: assertion is always true, perhaps remove parentheses?